From c59733fbb1dada56ca96c9798137c53fa2e88500 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Wed, 11 Dec 2019 17:22:38 -0800
Subject: [PATCH 047/322] [FIO toup] arch: arm: imx7ulp: fix rpmsg reservation
 conflict with OP-TEE

Move the rpmsg to reserved hole 0x9df00000.

Signed-off-by: Michael Scott <mike@foundries.io>
Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 arch/arm/dts/imx7ulp.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/imx7ulp.dtsi b/arch/arm/dts/imx7ulp.dtsi
index ff62156f75d..63bd21838b1 100644
--- a/arch/arm/dts/imx7ulp.dtsi
+++ b/arch/arm/dts/imx7ulp.dtsi
@@ -64,9 +64,9 @@
 			linux,cma-default;
 		};
 
-		rpmsg_reserved: rpmsg@9FFF0000 {
+		rpmsg_reserved: rpmsg@9DF00000 {
 			no-map;
-			reg = <0x9FF00000 0x100000>;
+			reg = <0x9DF00000 0x100000>;
 		};
 
 	};
-- 
2.34.1

